elastic/system/ksp-allow-elastic-process-cat-java.yaml

```diff
@@ -0,0 +1,26 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-allow-elastic-process-cat-java
+  namespace: default                       #change label to match your label
+spec:
+  tags: ["elastic", "jdk", "java"]
+  message: "cat process performed by elastic binary"
```

Comment:
policy is not only for cat process. @AbhinavCSY Can you think more suitable message?